### PR TITLE
Master rename error message bst

### PR DESCRIPTION
**Pull request by tde-banana-odoo**
Task: https://www.odoo.com/web#id=47042&view_type=form&model=project.task&menu_id=5200

### CODE REVIEW

File: addons/hr_expense/models/hr_expense.py [hunk 1]
```diff
@@ -516,14 +516,14 @@ def _check_payment_mode(self):
         for sheet in self:
             expense_lines = sheet.mapped('expense_line_ids')
             if expense_lines and any(expense.payment_mode != expense_lines[0].payment_mode for expense in expense_lines):
-                raise ValidationError(_("Expenses must have been paid by the same entity (Company or employee)"))
+                raise ValidationError(_("Expenses must be paid by the same entity (Company or employee)"))
```
Comment (bst-odoo) [inline addons/hr_expense/models/hr_expense.py]: I forgot the . at the end of the sentence : "Expenses must be paid by the same entity (Company or employee)."

File: addons/hr_holidays/models/hr_leave.py [hunk 2]
```diff
@@ -382,7 +382,7 @@ def action_approve(self):
               and not self.env.user.has_group('hr_holidays.group_hr_holidays_manager'):
                 raise UserError(_('You must be %s manager to approve this leave') % (holiday.employee_id.name))
             elif validation_type == 'hr' and not self.env.user.has_group('hr_holidays.group_hr_holidays_manager'):
-                raise UserError(_('You must be a Human Resource Manager to approve this Leave'))
+                raise UserError(_('You must be a Human Resource Manager to approve this leave'))
```
Comment (bst-odoo) [inline addons/hr_holidays/models/hr_leave.py]: ==> 'You must be a Human Resource Manager to approve this leave.'

. is missing at the end of the sentence.

File: addons/point_of_sale/models/pos_config.py [hunk 3]
```diff
@@ -239,7 +239,7 @@ def _check_company_invoice_journal(self):
     @api.constrains('company_id', 'journal_ids')
     def _check_company_payment(self):
         if self.env['account.journal'].search_count([('id', 'in', self.journal_ids.ids), ('company_id', '!=', self.company_id.id)]):
-            raise ValidationError(_("The company of a payment method is different than the one of point of sale"))
+            raise ValidationError(_("The method payments and the point of sale must belong to the same company"))
```
Comment (bst-odoo) [inline addons/point_of_sale/models/pos_config.py]: Once again, a . is missing : "The method payments and the point of sale must belong to the same company."

File: addons/point_of_sale/models/pos_config.py [hunk 4]
```diff
@@ -224,12 +224,12 @@ def _compute_current_session_user(self):
...
     def _check_company_location(self):
         if self.stock_location_id.company_id and self.stock_location_id.company_id.id != self.company_id.id:
-            raise ValidationError(_("The company of the stock location is different than the one of point of sale"))
+            raise ValidationError(_("The stock location and the point of sale must belong to the same company"))
 
     @api.constrains('company_id', 'journal_id')
     def _check_company_journal(self):
         if self.journal_id and self.journal_id.company_id.id != self.company_id.id:
-            raise ValidationError(_("The company of the sales journal is different than the one of point of sale"))
+            raise ValidationError(_("The sales journal and the point of sale must belong to the same company"))
```
Comment (bst-odoo) [inline addons/point_of_sale/models/pos_config.py]: Once again, a . is missing : "The sales journal and the point of sale must belong to the same company."

File: addons/purchase/models/stock.py [hunk 5]
```diff
@@ -105,7 +105,7 @@ def _get_buy_pull_rule(self):
             buy_route_id = self.env['stock.location.route'].search([('name', 'like', _('Buy'))])
             buy_route_id = buy_route_id[0].id if buy_route_id else False
         if not buy_route_id:
-            raise UserError(_("Can't find any generic Buy route."))
+            raise UserError(_("Can't find any generic Buy route. Please create a route with the 'Buy' action for your receipts operation types."))
```
Comment (bst-odoo) [inline addons/purchase/models/stock.py]: Remove the word "generic" : "Can't find any Buy route. Please create a route with the 'Buy' action for your receipts operation types."

File: addons/survey/models/survey.py [hunk 6]
```diff
@@ -684,7 +684,7 @@ class SurveyLabel(models.Model):
     def _check_question_not_empty(self):
         """Ensure that field question_id XOR field question_id_2 is not null"""
         if not bool(self.question_id) != bool(self.question_id_2):
-            raise ValidationError(_("A label must be attached to one and only one question"))
+            raise ValidationError(_("A label must be attached to only one question"))
```
Comment (bst-odoo) [inline addons/survey/models/survey.py]: Missing . ==> "A label must be attached to only one question."

File: addons/survey/models/survey.py [hunk 7]
```diff
@@ -803,7 +803,7 @@ class SurveyUserInputLine(models.Model):
     def _answered_or_skipped(self):
         for uil in self:
             if not uil.skipped != bool(uil.answer_type):
-                raise ValidationError(_('A question cannot be unanswered and skipped'))
+                raise ValidationError(_('This question cannot be unanswered or skipped'))
```
Comment (bst-odoo) [inline addons/survey/models/survey.py]: Missing . ==> "This question cannot be unanswered or skipped."

File: odoo/addons/base/models/res_lang.py [hunk 8]
```diff
@@ -225,7 +225,7 @@ def write(self, vals):
             raise UserError(_("Language code cannot be modified."))
         if vals.get('active') == False:
             if self.env['res.users'].search([('lang', 'in', lang_codes)]):
-                raise UserError(_("Cannot unactivate a language that is currently used by users."))
+                raise UserError(_("Cannot disactivate a language that is currently used by users."))
```
Comment (bst-odoo) [inline odoo/addons/base/models/res_lang.py]: Really?!? I wrote disactivate ? shame on me . It should be deactivate.

File: addons/hr_holidays/models/hr_leave.py [hunk 9]
```diff
@@ -413,8 +413,8 @@ def action_approve(self):
             if (validation_type in ['hr', 'both']) and (manager and manager != current_employee)\
               and not self.env.user.has_group('hr_holidays.group_hr_holidays_manager'):
                 raise UserError(_('You must be %s manager to approve this leave') % (holiday.employee_id.name))
-            elif validation_type == 'manager' and not self.env.user.has_group('hr_holidays.group_hr_holidays_manager'):
-                raise UserError(_('You must be a Human Resource Manager to approve this Leave'))
+            elif validation_type == 'hr' and not self.env.user.has_group('hr_holidays.group_hr_holidays_manager'):
```
Comment (Xavier-Do) [inline addons/hr_holidays/models/hr_leave.py]: Le changement de 'manager' à 'hr' est voulu ou c'est un fail durant le rebase?

File: addons/project/models/project.py [hunk 10]
```diff
@@ -84,8 +84,12 @@ def get_alias_values(self):
...
         for project in self:
             if project.tasks:
-                raise UserError(_('You cannot delete a project containing tasks. You can either delete all the project\'s tasks and then delete the project or simply deactivate the project.'))
-        return super(Project, self).unlink()
+                raise UserError(_('You cannot delete a project containing tasks. You can either archive it or first delete all of its tasks.'))
+            if project.analytic_account_id and not project.analytic_account_id.line_ids:
+                analytic_accounts_to_delete |= project.analytic_account_id
+        res = super(Project, self).unlink()
+        analytic_accounts_to_delete.unlink()
+        return res
```
Comment (Xavier-Do) [inline addons/project/models/project.py]: Même commentaire ici, changement de logique, je présume que ce n'est pas voulu?

File: addons/hr_holidays/models/hr_leave.py (hunk 9 above)
Comment (bst-odoo) [reply addons/hr_holidays/models/hr_leave.py]: Romain a fait des changements sur les droits d'accès HR. Autant ne pas y toucher.

**Pull request merged**